app/src/main/java/com/futurice/freesound/feature/audio/ExoPlayerStateObservable.java

```diff
@@ -56,8 +56,8 @@ protected void subscribeActual(final Observer<? super ExoPlayerState> observer)
         observer.onSubscribe(listener);
         exoPlayer.addListener(listener);
         if (emitInitial) {
-            emitValue(ExoPlayerState.create(exoPlayer.getPlayWhenReady(),
-                                            exoPlayer.getPlaybackState()),
+            emitValue(new ExoPlayerState(exoPlayer.getPlayWhenReady(),
```

Comment:
I get a warning about `this.exoPlayer` being `@NonNull` but not initialized by `ExoPlayerStateObservable`'s package visible constructor. Do you get the same?